standard-app/templates/configs/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+# Default configMap
+{{- if .Values.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-configmap
+data:
+{{- range $key, $value := .Values.configMap }}
+  {{ $key }}: {{ $value }}
+{{- end }}
+---
+{{- end }}
+
+# Deployment configMap
+{{- if .Values.apps.configMap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-configmap
+data:
+{{- range $key, $value := $appConfig.configMap }}
+  {{ $key }}: {{ $value }}
+{{- end }}
+---
+{{- end }}
+
+# Script configMap
+{{- if .Files.Exists "scripts" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-scripts
+data:
+  {{- range $path, $content := .Files.Glob "scripts/*" }}
+  {{- $filename := path.Base $path }}
+  {{- $ext := path.Ext $filename }}
+  {{- if or (eq $ext ".sh") (eq $ext ".js") }}
+  {{ $filename }}: |-
+{{ $content | indent 4 }}
+  {{- end }}
+  {{- end }}
+{{- end }}
```

Comment:
we don't really need that: you can template all paths in the values file
please see https://github.com/cloudkite-io/infrastructure/blob/monitoring/kubernetes/helm/grafana-operator/prod.values.yaml#L31-L36 and https://github.com/cloudkite-io/infrastructure/blob/monitoring/kubernetes/helm/grafana-operator/prod.values.yaml#L31-L36

Reply:
my bad, here is the correct one https://github.com/cloudkite-io/infrastructure/blob/monitoring/kubernetes/helm/grafana-operator/templates/grafana/dashboard.yaml#L11

Reply:
my bad, here is the correct one https://github.com/cloudkite-io/infrastructure/blob/monitoring/kubernetes/helm/grafana-operator/templates/grafana/dashboard.yaml#L11